multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/variables/Variables.java

```diff
@@ -822,6 +823,10 @@ public List<CloudApplication> getDefaultValue() {
                                                                                                     })
                                                                                                     .defaultValue(Collections.emptyList())
                                                                                                     .build();
+    Variable<LogsOffset> LOGS_OFFSET_FOR_APP_EXECUTION = ImmutableJsonStringVariable.<LogsOffset> builder()
```

Comment:
I can see that `LogsOffset` implements Serializable which means that this variable can be persisted in db as SimpleVariable. It can be leave as it is now.

Reply:
Used ImmutableSimpleVariable, so it can be stored with a default serialization and consistent to the other way